src/components/OnBoarding/ThirdForm.jsx

```diff
@@ -0,0 +1,316 @@
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  Button,
+  Checkbox,
+  Text,
+  Box,
+  Flex,
+  FormLabel,
+  FormControl,
+  Input,
+  Stack,
+  Select,
+  Alert,
+  AlertIcon,
+  Link,
+  AlertTitle,
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverHeader,
+  PopoverBody,
+  PopoverArrow,
+  PopoverCloseButton,
+  HStack,
+  Image,
+} from '@chakra-ui/react';
+import FPH_LOGO from './fph_logo.png';
+
+const ThirdForm = ({ handleChange, prevStep, handleSubmit }) => {
+  const daysOfWeek = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
+  const [businessHours, changeBusinessHours] = useState(
+    daysOfWeek.reduce(
+      (acc, day) => ({
+        ...acc,
+        [day]: {
+          start: { hour: '9', minute: '00', ampm: 'AM' },
+          end: { hour: '5', minute: '00', ampm: 'PM' },
+        },
+      }),
+      {},
+    ),
+  );
+
+  const TimeInput = ({ label, value, onChange, isReadOnly }) => (
+    <Stack direction="row" align="center">
+      <FormLabel mb="0">{label}</FormLabel>
+      <Select
+        value={value.hour}
+        onChange={e => onChange({ ...value, hour: e.target.value })}
+        w="auto"
+        disabled={isReadOnly}
+      >
+        {[...Array(12).keys()].map(h => (
+          <option key={h} value={h + 1}>
+            {h + 1}
+          </option>
+        ))}
+      </Select>
+      <Select
+        value={value.minute}
+        onChange={e => onChange({ ...value, minute: e.target.value })}
+        w="auto"
+        disabled={isReadOnly}
+      >
+        {['00', '15', '30', '45'].map(m => (
+          <option key={m} value={m}>
+            {m}
+          </option>
+        ))}
+      </Select>
+      <Select
+        value={value.ampm}
+        onChange={e => onChange({ ...value, ampm: e.target.value })}
+        w="auto"
+        disabled={isReadOnly}
+      >
+        {['AM', 'PM'].map(ampm => (
+          <option key={ampm} value={ampm}>
+            {ampm}
+          </option>
+        ))}
+      </Select>
+    </Stack>
+  );
+
+  TimeInput.propTypes = {
+    label: propTypes.string.isRequired,
+    value: propTypes.shape({
+      hour: propTypes.string.isRequired,
+      minute: propTypes.string.isRequired,
+      ampm: propTypes.string.isRequired,
+    }).isRequired,
+    onChange: propTypes.func.isRequired,
+    isReadOnly: propTypes.bool.isRequired,
+  };
+
+  const handleTimeChange = (day, period, value) => {
+    changeBusinessHours({ ...businessHours, [day]: { ...businessHours[day], [period]: value } });
+  };
+
+  const [phoneNumber, changePhoneNumber] = useState('');
+  const [heardAbout, changeHeardAbout] = useState('');
+  const [termsAndConditionsIsOpened, changeTermsAndConditionsIsOpened] = useState(false);
+  const [termsAndConditionsAccepted, changeTermsAndConditionsAccepted] = useState(false);
+  const [submitAndAcceptedFalse, changeSubmitAndAcceptedFalse] = useState(false);
+  console.log(submitAndAcceptedFalse);
```

Comment:
dont create these states, use the states from `formData` in here, such that when we press the back button, we don't lose all of our entries